functional-tests/src/main/scala/com/gu/membership/tags/Tags.scala

```diff
@@ -2,7 +2,23 @@ package com.gu.membership.tags
 
 import org.scalatest.Tag
 
+/* Usage: The tag is added as a comma separated list for each scenario after the scenario name
```

Comment:
:tada: Thanks for adding these steps! Mind adding it into the `README` in `functional-tests/` as well?